aiorun.py

```diff
@@ -25,6 +25,11 @@
 __version__ = '2018.9.1'
 logger = logging.getLogger('aiorun')
 
+try:
+    from asyncio import all_tasks
```

Comment:
Add a comment above this line to specify that Python 3.7 added `all_tasks`. It'll be useful a few years from now when we drop support for 3.6, and this try-except can be refactored into a simple import.